.github/renovate.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "automerge": true,
+  "schedule": [
+    "every weekend"
+  ],
+  "packageRules": [
```

Comment:
We have already tested this feature, It is good but at the same time too much noise(creating many PR's and it will keep waiting for 3 days).  Let's remove this stabilityDays for now.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/767/commits/6e7b26a3acfdc12a3aaa84a91076eed2dcc6dd7c